code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -221,6 +221,19 @@
 		M.Sleeping(30)
 	M.drowsyness = max(M.drowsyness, 30)
 
+/datum/reagent/mothpheromones
+	name = "Moth pheromones"
+	id = "mothpheromones"
+	description = "Basically the liquid concept of fluttering towards a lamp."
+	reagent_state = LIQUID
+	color = "#8fddf7" // rgb: 143, 221, 247
+	taste_description = "silk"
+
+/datum/reagent/mothpheromones/on_mob_life(mob/living/M, alien)
+	if(alien == IS_MOTH && prob(7))
+		M.emote(pick("twitch","drool","moan","gasp"))
```

Comment:
remove moan

Reply:
No

Reply:
that makes this easy then

Reply:
Remove the moan for Qust to approve it, so we have nice lamp bars, please.